### PR TITLE
Feat: invalidate oversized files

### LINKED ISSUE #569: Validate file size for uploaded files
## Description
Right now there is no validation for uploaded files size in the API, we should be able to have a configurable value for the max size allowed. Part of this work should be to indentify all the endpoints that allow file upload.

### PR DESCRIPTION
**Pull request by jodiezhuu**
## Description
Validate file upload size before API call at each upload file endpoint

### Related Issues
Fixes #569

### CODE REVIEW

**Review by josebui — APPROVED** [review]
Looks good

File: terraso_backend/apps/storage/file_utils.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/.
+
+def is_file_upload_oversized(files, max_size):
+    for file in files:
+        return get_file_size(file) > max_size
```
Comment (paulschreiber) [inline terraso_backend/apps/storage/file_utils.py:18]: suggestion: `get_file_size` is unnecessary. Reference `file.size` directly.

File: terraso_backend/apps/storage/file_utils.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/.
+
+def is_file_upload_oversized(files, max_size):
+    for file in files:
```
Comment (paulschreiber) [inline terraso_backend/apps/storage/file_utils.py:17]: question: what happens if there is more that one file? what if the second file is oversize but the first one is not? what is the expected behaviour here?

File: terraso_backend/apps/storage/file_utils.py (hunk 1 above)
Comment (jodiezhuu) [reply terraso_backend/apps/storage/file_utils.py:18]: @paulschreiber We need the get_file_size function to mock an oversized file when unit testing: Check `terraso_backend/tests/shared_data/test_views.py`

**Pull request merged**